x11/xorg-scripts/files/apple-silicon.diff

```diff
@@ -0,0 +1,19 @@
+--- config.guess.orig	2020-09-07 13:00:56.000000000 -0400
++++ config.guess	2020-09-07 13:01:18.000000000 -0400
+@@ -3,7 +3,7 @@
+ #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
+ #   2000, 2001, 2002, 2003 Free Software Foundation, Inc.
+ 
+-timestamp='2003-06-17'
++timestamp='2020-09-07'
```

Comment:
(1) Technically this part of the patch is unnecessary. 

(2) Have you tried submitting this patch upstream? Always best for the upstream to integrate things like this so that our MP patches are minimized.